CONTRIBUTING.md

```diff
@@ -81,6 +82,55 @@ $ npx prettier * --write
 
 This should be done before opening a PR to keep checked-in code consistent.
 
+### Onboarding
+
+To onboard your release tests (Dashboards plugins) onto this repo test function, you can follow the next steps.
+
+0. Create a name to identify your plugin, e.g `plugin-name`
+
+The dir name shall be descriptive to identify your plugin. You can use the same name defined in build repo https://github.com/opensearch-project/opensearch-build/tree/main/scripts/components
+
+1. Move test files under `cypress/integration/plugins/<plugin-name>`
+
+2. Move fixtures under `cypress/fixtures/plugins/<plugin-name>`
+
+3. Move custom commands under `cypress/fixtures/plugins/<plugin-name>`
+
+4. Move constants to `cypress/utils/constants.js`
+
+5. Run tests
+
+Start OpenSearch and OpenSearch Dashboards. Then refer to this test execution command https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/integtest.sh#L89
```

Comment:
I'm afraid this link may go out of date once changes are made to that workflow. Maybe remove and just keep the example like you have below?

Reply:
Good callout. Let me remove the line number reference. I still want to keep a reference to the file since it is what infra build will use.